src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -103,9 +103,13 @@ export function DownloadButton(props) {
 DownloadButton.propTypes = {
     tbl_id      : PropTypes.string,
     tbl_grp     : PropTypes.string,
+    need_selectInfo:PropTypes.bool
```

Comment:
Minor:  Can you change the property to `checkSelectedRow`?

Reply:
Yes, I will change it.